templates/_components/free_text.html.twig

```diff
@@ -0,0 +1,23 @@
+<div class="govuk-width-container">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-half" >
+            <div class="govuk-border-right-free-text" style="height:110px ; padding-left:50px ; margin-left:80px ">
```

Comment:
This hard coded height should be removed as it's going to cause issues with display if too much text is entered.